server/test/auth.spec.js

```diff
@@ -0,0 +1,89 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const { server } = require("../app.js");
+const dotenv = require('dotenv')
+const expect = require("chai").expect;
+
+dotenv.config();;
+
+chai.should();
+chai.use(chaiHttp);
+
+const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjYxMjdmYjA3OTNhYTZmM2ExYzkyM2ZiZiIsImlhdCI6MTYzMDAxMDExOSwiZXhwIjoxNjMwNjE0OTE5fQ.EE8omcwEkQkCEJUEMyY2ODQgBHOZk3g5spRPwQ6jynM';
+
+
+// Test for login
+describe("/POST login", () => {
+  it("it should  return 200 if credentials are valid or 401 if not", (done) => {
+    chai
+      .request(server)
+      .post(`/auth/login`)
+      .send({ email: "jondoe@vmail.com", password: "12345678" })
+      .end((err, res) => {
+        expect(res.status).to.be.oneOf([200, 401]);
+        done();
+      });
```

Comment:
This test case makes no sense, you are testing if it's successful or not but you are passing correct credentials? Just separate and test for a successful case and a unsuccessful case.